manifests/params.pp

```diff
@@ -10,7 +10,7 @@
       $package_name_v4  = 'bird'
       # Compatibility for Debian oldstable and Ubuntu old LTS
       # the bird package now provides bird and bird6 daemons
-      if ($facts['os']['name'] == 'ubuntu' and $facts['lsbdistcodename'] in ['precise', 'trusty']) or
+      if ($facts['os']['name'] == 'ubuntu' and $facts['os']['distro']['release'] in ['precise', 'trusty']) or
```

Comment:
We've dropped support for Ubuntu Precise and Debian 7. Ubuntu Trusty can also be dropped since it's EOL. Maybe a good chance to clean it up? I'd also accept a PR that drops Debian 8 and moves it all to static parameters in init.pp.

Reply:
I will move it to init.pp in another PR